src/runtime/runtime_rpi3.go

```diff
@@ -0,0 +1,120 @@
+// +build rpi3
+package runtime
+
+import (
+	dev "device/arm64"
```

Comment:
Why `dev` here and `rpi` in the machine package?
Please keep it consistent. I wouldn't have renamed this package locally but with the package name being `aarch64` (and not `bcm283x` for example) that might look strange.